relay/kusama/src/paras_scheduler_migration.rs

```diff
@@ -104,6 +104,7 @@ pub mod v1 {
 		ValueQuery,
 	>;
 
+	#[allow(dead_code)]
 	#[allow(deprecated)]
 	pub type MigrateToV1<T> = VersionedMigration<
```

Comment:
Instead of this, I can also remove this file completely, but thought it might be useful for future reference.. lemme know if you think otherwise.

Reply:
I'd just remove it. We have git history for future reference.